### ISSUE TITLE
Enable editing properties by keyboard only

### ISSUE THREAD

**Issue by rainman110** [dlr-gtlab/gtlab-core#36]
In GitLab by @otte_to on Feb 21, 2020, 10:37

## What is the purpose of the change?
-  There should be a way to navigate through the properties using the keyboard only.
- When setting up a model, all input vals are typed by keyboard.
- Using the mouse to select the next property is not very user friendly.

- The "arror down" seems to do the job, but as soon as a dropdown menu occurs, it changes its behaviour and selects entries from the dropdown menu instead of moving to the next property. 

This should be consistent for best user experience and efficiency.

Suggestion:

## Which tasks have to be performed?
- [ ] The "Tab" key should move the cursor the the next field no matter if there is a dropdown menu or group.

## Info: This issue was submitted to MANTIS (0020476) in 07/2016

**Comment by jensschmeink** [comment]
After a talk with the author of the issue the problem is not recognized anymore
